### ISSUE TITLE
Plugin stops Strapi 4.15.1 from building

### ISSUE THREAD

**Issue by xenobytezero** [ComfortablyCoding/strapi-plugin-publisher#124]
Adding the plugin to my Strapi 4.15.1 app causes the build to fail

```
ERROR in ./node_modules/strapi-plugin-publisher/admin/src/components/Action/Action.js 3:0-80
Module not found: Error: Package path ./admin/src/content-manager/utils is not exported from package <APP_ROOT>\node_modules\@strapi\admin (see exports field in <APP_ROOT>\node_modules\@strapi\admin\package.json) 
```

In 4.15.1 the `@strapi/admin` package added an export map to it's `package.json`, meaning that direct source imports don't work any more, so the following line breaks in `admin/src/components/Action/Action.js`

```ts
import { createYupSchema } from '@strapi/admin/admin/src/content-manager/utils';
```

It doesn't look like `createYupSchema` is exposed properly, so the schema validation imight need to be disabled?

**Comment by ComfortablyCoding** [comment]
I'll look into directly adding the code for this to the plugin so it's no longer dependant on a path.

P.S.
`4.15.0` and `4.15.1` are deprecated versions. The current latest active version is `4.15.2`

**Comment by xenobytezero** [comment]
Sorry

> I'll look into directly adding the code for this to the plugin so it's no longer dependant on a path.
> 
> P.S. `4.15.0` and `4.15.1` are deprecated versions. The current latest active version is `4.15.2`

Ahh yea sorry, I updated to 14.5.2 and saw the issue, the issue was introduced in the 14.5.1 release.

**Comment by 101sam** [comment]
Any updates?

**Comment by ComfortablyCoding** [comment]
This should be resolved in the latest release (1.5.3)
